### PR TITLE
feat: Jenkinsfile

### PR DESCRIPTION
**Pull request by psturc**
https://issues.jboss.org/browse/AEROGEAR-9110

### CODE REVIEW

**Comment by aliok** [comment]
@psturc is this ready to verify?

The Jenkins checks in GH look good. Any further verification needed?

**Comment by psturc** [comment]
@aliok yes, sorry, I forgot to update the ticket after final change. It's ready to verify/review.

File: Jenkinsfile [hunk 1]
```diff
@@ -0,0 +1,167 @@
+pipeline {
+    agent {
+        node {
+            label "psi_rhel7_openshift311"
+        }
+    }
+
+    libraries {
+        lib('fh-pipeline-library')
+        lib('qe-pipeline-library')
```
Comment (psturc) [inline Jenkinsfile:10]: https://github.com/aerogear/qe-pipeline-library

**Comment by aliok** [comment]
Looks good. Feel free to merge

**Pull request merged**